README.md

```diff
@@ -235,6 +235,7 @@ will precede the `go` command to start thinking with `sd 5`. The other `go_comma
     -rated
     -casual
 ```
+  - `blocklist`: An indented list of usernames from which the challenges are always declined. If this option is not present, then the list is considered empty.
```

Comment:
This should be `block_list` to match the config file. Same in the matchmaking section.